### PR TITLE
10179 add module elico_git and divide it in 2 modules

### PR DESCRIPTION
**Pull request by Rona111**
add module elico_git and divide it in 2 modules: project_task_git_branch and project_git_branch.

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: project_git_branch/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Project Git Branch
+==================
+
```
Comment (elicoidal) [inline project_git_branch/README.rst:8]: add:
This modules extends the project object and add some information about the 
Git branch where the code is located

File: project_git_branch/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20project_git_branch%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Chen Rong <chen.rong@elico-corp.com>
```
Comment (elicoidal) [inline project_git_branch/README.rst:50]: add Augustin

File: project_git_branch/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
```
Comment (elicoidal) [inline project_git_branch/README.rst]: replace wit Elico

File: project_task_git_branch/README.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Task GIT Branch
+===============
+
+This module allows you to manage your git .
```
Comment (elicoidal) [inline project_task_git_branch/README.rst:9]: replace:
This module allows you to manage your git .
with
This module allows you to manage your git within the tasks of a project.

File: project_task_git_branch/project_view.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    <record id="project_task_git_form" model="ir.ui.view">
+        <field name="name">project.task.git.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Git">
+                    <group col="2">
+                        <field name="git_branch" />
+                        <field name="git_environment" on_change="onchange_gsb(project_id, git_environment)"/>
```
Comment (elicoidal) [inline project_task_git_branch/project_view.xml:13]: PEP8

**Comment by elicoidal** [comment]
minor modifications. Once corrected can be merged

**Comment by siyuancn** [comment]
Please user the structure:
A module is organised in a few directory:

```
controllers/: contains controllers (http routes)
data/: data xml
demo/: demo xml
models/: models definition
report/: reporting models (BI/analysis), Webkit/RML print report templates
static/: contains the web assets, separated into css/, js/, img/, lib/, ...
views/: contains the views and templates, and QWeb report print templates
wizards/: wizard model and views
```

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**